delatore/sources/awx_api.py

```diff
@@ -109,3 +120,12 @@ def single_template_filter(template_name: str):
     if template_name:
         return {'name__iexact': template_name}
     return None
+
+
+def create_record(data: Dict, index: int):
```

Comment:
Make it `_protected` if you don't use it anywhere else (in this case pydoc is not required)